docker/java/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'java'
+apply plugin: 'application'
+
+mainClassName = 'Main'
+
+repositories {
+  mavenCentral()
+}
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+  implementation 'io.nats:jnats:2.15.5'
+}
+
+jar {
+  manifest {
+    attributes (
+      'Main-Class': "Main"
+    )
+  }
+}
```

Comment:
@bruth I think you need to fully qualify the class name with it's package.

```
jar {
    manifest {
        attributes 'Main-Class': 'example.MainClass'
    }
}
```

Reply:
Thanks, let me try that.

Reply:
:disappointed: Same error, but noting the change in the message `example.Main`. So it appears to be looking for that.. is the directory structure correct? Under `src` is copied `main/java/example/Main.java`? Also is the `MANIFEST.MF` necessary with the attributes in the build.gradle file?

Reply:
See my next comment...